js/picture-form.js

```diff
@@ -0,0 +1,98 @@
+import { sendData } from './fetch-api.js';
+import { resetUploadForm } from './picture-upload.js';
+import { isEscEvent, toggleModal } from './util.js';
+
+const main              = document.querySelector('main');
+const successPopup      = document.querySelector('#success').content.querySelector('.success');
+const failPopup         = document.querySelector('#error').content.querySelector('.error');
+const pictureEditForm   = document.querySelector('.img-upload__overlay');
+
+const successButton     = successPopup.querySelector('.success__button');
+const failButton        = failPopup.querySelector('.error__button');
+
+const removeSuccessModal = () => {
+  main.removeChild(successPopup);
+  successButton.removeEventListener('click', deleteSuccessPopup);
+  document.removeEventListener('keydown', deleteSuccessPopupEsc);
+  window.removeEventListener('click', deleteSuccessPopupOut);
+}
+
+const removeFailModal = () => {
+  main.removeChild(failPopup);
+  failButton.removeEventListener('click', deleteFailPopup);
+  document.removeEventListener('keydown', deleteFailPopupEsc);
+  window.removeEventListener('click', deleteFailPopupOut);
+}
+
+const setUserFormSubmit = (evt) => {
+  evt.preventDefault();
+
+  sendData(
+    () => {
+      toggleModal(pictureEditForm);
+      resetUploadForm();
+
+      successPopup.cloneNode(true);
```

Comment:
добавлять нужно новый узел
то же для failPopup
```suggestion
      const node = successPopup.cloneNode(true);
```